Mathlib/RepresentationTheory/Rep.lean

```diff
@@ -57,15 +57,25 @@ instance (V : Rep k G) : Module k V := by
   change Module k ((forget₂ (Rep k G) (ModuleCat k)).obj V)
   infer_instance
 
+-- TODO: is there some abstract machinery for this?
+/-- `ModuleCat.Hom.hom` as an isomorphism of monoids. -/
+@[simps]
+def homMulEquiv (V : ModuleCat k) : End V ≃* (V →ₗ[k] V) where
+  toFun := ModuleCat.Hom.hom
+  invFun := ModuleCat.asHom
+  map_mul' _ _ := rfl
+  left_inv _ := rfl
+  right_inv _ := rfl
```

Comment:
Is there a categorical name for this thing?

Reply:
It seems to me like this should work for endomorphism monoids in any `bundled` category.